recipe/meta.yaml

```diff
@@ -1,34 +1,33 @@
-{% set version = "2.16.3" %}
+{% set version = "2.14.6" %}
 
 package:
   name: pydantic-core
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pydantic-core/pydantic_core-{{ version }}.tar.gz
-  sha256: 1cac689f80a3abab2d3c0048b29eea5751114054f032a941a32de4c852c59cad
+  sha256: 1fd0c1d395372843fba13a51c28e3bb9d59bd7aebfeb17358ffaaa1e4dbbe948
 
 build:
+  script_env:
+    - RUSTFLAGS=-C link-args=-headerpad_max_install_names  # [osx and x86_64]
```

Comment:
```suggestion
    # on osx-64, we get an error from install_name_tool: changing install names or rpaths can't be redone for: <path> (for architecture x86_64) because larger updated load commands do not fit (the program must be relinked, and you may need to use -headerpad or -headerpad_max_install_names)
    # After some investigation, rustc doesn't seem to read our LDFLAGS
    - RUSTFLAGS=-C link-args=-headerpad_max_install_names  # [osx and x86_64]
```